CONTRIBUTING.md

```diff
@@ -65,8 +65,26 @@ organize them into different categories. For new contributors, we recommend
 starting with issues labeled "good first issue." If you're interested in
 enhancing our documentation, you can filter issues using the "documentation"
 label to find issues specifically related to documentation improvement.
-  
+
 Once you have decided which issue to work on, you can express your interest by
 posting a comment on it. When you submit your proposed fix for an issue, link
 your PR to the issue with one of the supported
 [keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
+
+## Development environment
+
+To set up a development environment to run from source,
+you can install it in a virtual environment,
+for example:
+
+```
+virtualenv venv
+source venv/bin/activate
+pip install -e .
+
+# setuptools is required to avoid `ModuleNotFoundError: No module named 'pkg_resources'`
+pip install setuptools
```

Comment:
This is only true on newer Python 3.12, not 3.10. Can you make not for it or maybe we can add setuptools to requirements, it should be ok even in Python 3.10.

Reply:
not sure, maybe we can add it to requirements already... or add specific version to tox, so it will not work on any other python version. Only 3.10

Reply:
I think you can... We mention in setup.cfg that `python_requires = >=3.10` and `Programming Language :: Python :: 3.10`.
I think that we should add 3.12 and simply add setuptools as requirements.